scripts/examples/typescript/README.md

```diff
@@ -0,0 +1,41 @@
+# A Typescript bindings for XenAPI
+A Typescript bindings for XenAPI, which is inspired by [Python XenAPI](https://xapi-project.github.io/xen-api/usage.html).
+
+## Usage
+Install lib from npm repository
+```
+$ npm install xen-api-ts
+```
+
+The usage of Typescript XenAPI is almost identical to the Python XenAPI, except it's asynchronous and requires async/await.
+```
+import { xapi_client } from "xen-api-ts";
```

Comment:
nit but if you're giving an MVP example you should have `import 'dotenv/config` if using ES6 or `require('dotenv').config()`

Reply:
there is a npm script to import dotenv,  
```
"scripts": {
     "test": "node -r ts-node/register -r dotenv/config ",
     "build": "tsc"
},
```